helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloLogService.java

```diff
@@ -58,12 +64,26 @@
     this.logStreamProvider = logStreamProvider;
   }
 
+  private static <T> T get(Future<T> future)
+      throws InterruptedException, ExecutionException, TimeoutException {
+    return future.get(1, TimeUnit.SECONDS);
+  }
+
+  @Override
+  protected ScheduledExecutorService executor() {
+    final ThreadFactory threadFactory = new ThreadFactoryBuilder()
+        .setNameFormat(serviceName())
+        .setDaemon(false)
```

Comment:
Shouldn't this be set to true as daemon threads do not prevent the JVM from exiting when the program finishes.